src/General/Button/Button.test.tsx

```diff
@@ -15,6 +15,25 @@ it('<Button> should render with text "click me" and an onClick handler', () => {
   expect(ButtonSnapshot).toMatchSnapshot();
 });
 
+it('<Button> should render as button by default', () => {
+  const tag = 'a';
+  const { container } = render(<Button tag={tag}>test</Button>);
```

Comment:
We shouldn't pass any tag to `Button` and the assertion should be:

`expect(container.querySelector(tag).tagName).toBe('BUTTON');`